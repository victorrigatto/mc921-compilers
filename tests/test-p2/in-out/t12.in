int main(int i) {
  if(i > 1)
    return 0;
  return 1;
}
