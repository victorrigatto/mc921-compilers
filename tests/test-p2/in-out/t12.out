Program:
    FuncDef:
        Type: int @ 1:1
        Decl: ID(name=main)
            FuncDecl:
                ParamList:
                    Decl: ID(name=i)
                        VarDecl:
                            Type: int @ 1:10
                VarDecl:
                    Type: int @ 1:1
        Compound: @ 1:17
            If: @ 2:3
                BinaryOp: > @ 2:6
                    ID: i @ 2:6
                    Constant: int, 1 @ 2:10
                Return: @ 3:5
                    Constant: int, 0 @ 3:12
            Return: @ 4:3
                Constant: int, 1 @ 4:10
